libparsec/crates/types/src/certif.rs

```diff
@@ -70,6 +59,34 @@ pub enum UnsecureSkipValidationReason {
     Test,
 }
 
+macro_rules! impl_base_load {
+    ($name:ident) => {
+        impl $name {
+            pub fn base_verify_and_load(
+                signed: &[u8],
+                author_verify_key: &VerifyKey,
+            ) -> DataResult<Self> {
+                let serialized = author_verify_key
+                    .verify(signed)
+                    .map_err(|_| DataError::Signature)?;
+
+                let result: Self = format_vx_load(serialized)?;
```

Comment:
Should settle to use `format_vx_load` (like it is here) or ` $crate::serialization::format_vx_load` (like it is in `base_unsecure_load` ?

given this macro is exposed with `pub(super) use`, I'd rather use `$crate::...` is it won't break if the caller module remove its `use serialzation::format_vx_load`

Reply:
[6731fa0](https://github.com/Scille/parsec-cloud/pull/7431/commits/6731fa051bb816f0bb76b1ff21d27139208df024)